### PR TITLE
[CDAP-14038] Remove docs & references to CDAP Tracker

### PR DESCRIPTION
**Pull request by kmeelu**
TODO: run Docs Build and Test (currently at concurrent build limit)
EDIT: Docs Build seems disabled, so running full RUT
RBT: https://builds.cask.co/browse/CDAP-RUT1676-1

@CuriousVini Deleted Metadata Management files. In viewing MetadataHttpHandler, it doesn't seem like tracker APIs have been added in; however, let me know if the new Metadata UI replaced the Tracker UI, in which case I'll update the Metadata UI page (rather than deleting it).

### CODE REVIEW

File: cdap-docs/reference-manual/source/http-restful-api/configuration.rst [hunk 1]
```diff
@@ -121,11 +121,6 @@ format, the output would be similar to (showing the first portion, reformatted t
         <value>org.apache.hadoop.yarn.ipc.HadoopYarnProtoRPC</value>
       <source>yarn-default.xml</source>
     </property>
-    <property>
-      <name>mapreduce.job.maxtaskfailures.per.tracker</name>
```
Comment (anwar6953) [inline cdap-docs/reference-manual/source/http-restful-api/configuration.rst]: i believe this is unrelated to CDAP Tracker.

Comment (kmeelu) [reply cdap-docs/reference-manual/source/http-restful-api/configuration.rst]: Yep, adding this back in.

**Review by anwar6953 — APPROVED** [review]

**Comment by CuriousVini** [comment]
Resolved conflicts in `cdap-docs/_common/common-build.sh`

**Comment by CuriousVini** [comment]
Rebased to latest release/6.0
build: https://builds.cask.co/browse/CDAP-DQB461-1

File: cdap-docs/developer-manual/source/metadata/metadata-ui.rst [hunk 2]
```diff
@@ -22,33 +22,19 @@ Introduction
...
-It allows you to perform impact and root-cause analysis, delivers an audit-trail for
-auditability and compliance, and allows you to preview data. Metadata management furnishes access to
+It allows you to perform impact and root-cause analysis while delivering an audit-trail for
+auditability and compliance. Metadata management furnishes access to
 structured information that describes, explains, and locates data, making it easier to
 retrieve, use, and manage datasets.
 
 Metadata management also allows users to update metadata for datasets and streams. Users can add,
-remove, and update tags and user properties directly in the UI. It allows users to set
+remove, and update tags and business properties directly in the UI. It allows users to set
```
Comment (CuriousVini) [inline cdap-docs/developer-manual/source/metadata/metadata-ui.rst]: is this change needed?

File: cdap-docs/developer-manual/source/metadata/metadata-ui.rst [hunk 3]
```diff
@@ -22,33 +22,19 @@ Introduction
 The CDAP Metadata UI ("metadata management") lets you see how data is flowing into and out
 of datasets, streams, and stream views.
 
-It allows you to perform impact and root-cause analysis, delivers an audit-trail for
```
Comment (anwar6953) [inline cdap-docs/developer-manual/source/metadata/metadata-ui.rst:25]: There are still some references to Cask Tracker in this file. See lines 6-10, for instance.

Comment (CuriousVini) [reply cdap-docs/developer-manual/source/metadata/metadata-ui.rst:25]: Not sure why it was added in the PR, but line 6 to 10 is not being used. Also verified that generated page from this file does not have any instances of 'cask tracker'

**Review by anwar6953 — APPROVED** [review]

**Comment by CuriousVini** [comment]
https://builds.cask.co/browse/CDAP-DQB461-4
squashing and merging.
